client/src/components/embedding/components/Opacities/components/Opacity/index.tsx

```diff
@@ -52,7 +47,10 @@ export default function Opacity({ name, value, handleChange }: Props) {
             sdsType="textField"
             value={value}
             onChange={(event) => {
-              handleChange(event, Number(event.target.value));
+              handleChange(
+                event as unknown as Event,
```

Comment:
and this... (it is what it is)

Reply:
if the above works, we can remove ` as unknown as Event`!

Reply:
The challenge is that onChange on StyledInputText and InputSlider both use the same handleChange but the event argument in StyledInputText is of ChangeEvent type so requires explicit cast to Event to match the expected argument type of handleChange